test/unit/db/repos/accounts.js

```diff
@@ -48,871 +50,1404 @@ describe('db', () => {
+					return db.accounts.list().then(data => {
+						const columnNames = _.map(db.accounts.cs.select.columns, column => {
+							return column.prop || column.name;
 						});
 
-						it('should return "balance" as "bigint"', () => {
-							return db.accounts
-								.list({}, ['balance'], { limit: 1 })
-								.then(data => {
-									expect(data[0].balance).to.be.a('string');
-								});
+						expect(data[0]).to.have.all.keys(columnNames);
+					});
+				});
+
+				it('should return only "address" if fields specify ["address"]', () => {
+					return db.accounts.list({}, ['address']).then(data => {
+						data.forEach(account => {
+							expect(account).to.have.all.keys(['address']);
 						});
+					});
+				});
 
-						it('should return "u_balance" as "bigint"', () => {
-							return db.accounts
-								.list({}, ['u_balance'], { limit: 1 })
-								.then(data => {
-									expect(data[0].u_balance).to.be.a('string');
-								});
+				it('should return only "address" and "isDelegate" if fields specify ["address", "isDelegate"]', () => {
+					return db.accounts.list({}, ['address', 'isDelegate']).then(data => {
+						data.forEach(account => {
+							expect(account).to.have.all.keys(['isDelegate', 'address']);
 						});
+					});
+				});
 
-						it('should return "rate" as "bigint"', () => {
-							return db.accounts.list({}, ['rate'], { limit: 1 }).then(data => {
-								expect(data[0].rate).to.be.a('string');
+				it('should skip any unknown field specified ["address", "unKnownField"]', () => {
+					return db.accounts
+						.list({}, ['address', 'unKnownField'])
+						.then(data => {
+							data.forEach(account => {
+								expect(account).to.have.all.keys(['address']);
 							});
 						});
+				});
 
-						it('should return "fees" as "bigint"', () => {
-							return db.accounts.list({}, ['fees'], { limit: 1 }).then(data => {
-								expect(data[0].fees).to.be.a('string');
-							});
-						});
+				describe('property names', () => {
+					it('should return "producedBlocks" column if "producedBlocks" is asked', () => {
+						let actualResult;
 
-						it('should return "rewards" as "bigint"', () => {
-							return db.accounts
-								.list({}, ['rewards'], { limit: 1 })
-								.then(data => {
-									expect(data[0].rewards).to.be.a('string');
+						return db.accounts.list({}, ['producedBlocks']).then(data => {
+							actualResult = data;
+
+							return db
+								.query(
+									'SELECT producedblocks::bigint AS "producedBlocks" FROM mem_accounts'
+								)
+								.then(result => {
+									expect(actualResult).to.eql(result);
 								});
 						});
+					});
 
-						it('should return "vote" as "bigint"', () => {
-							return db.accounts.list({}, ['vote'], { limit: 1 }).then(data => {
-								expect(data[0].vote).to.be.a('string');
-							});
-						});
+					it('should return "missedBlocks" column if "missedBlocks" is asked', () => {
```

Comment:
`is requested`

Reply:
These tests are removed in PR https://github.com/LiskHQ/lisk/pull/1572 due to renaming the "missedBlocks" column.